tests/test_formpack_internals.py

```diff
@@ -61,6 +61,46 @@ def test_to_xml_fails_when_null_labels():
     fp[0].to_xml()
 
 
+def test_groups_disabled():
+    scontent = {'content': {
+                   'survey': [
+                              {'type': 'text','name': 'n1', 'label': ['aa']},
+                              {'type': 'begin_group', 'name': 'nada'},
+                              {'type': 'note', 'name': 'n2', 'label': ['ab']},
+                              {'type': 'end_group'},
+                              ],
+                   'translations': ['en'],
+                   'translated': ['label'],
+                  }
+                }
+
+    (ga, gz) = [scontent['content']['survey'][nn] for nn in [1, 3]]
+    # verify that "ga" and "gz" variables point to group begin/end
+    assert ga['type'] == 'begin_group'
+    assert gz['type'] == 'end_group'
+    assert ga.get('disabled') == gz.get('disabled') == None
+
+    # verify values before setting "disabled=TRUE"
+    fp = FormPack(scontent, id_string='xx')
+    ss = [ss for ss in fp[0].sections.values()][0]
+    # only(?) way to access groups is in the hierarchy of a child question
+    n2_parent = ss.fields['n2'].hierarchy[-2]
+    group_class_repr = "<class 'formpack.schema.datadef.FormGroup'>"
+    assert repr(n2_parent.__class__) == group_class_repr
```

Comment:
how about `isinstance(n2_parent, formpack.schema.datadef.FormGroup)`? (if it doesn't cause a circular import)

Reply:
:+1: